System.IO.FileSystem/System.IO.FileSystem.nfproj

```diff
@@ -78,19 +82,19 @@
     <Content Include="packages.lock.json" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="mscorlib, Version=1.15.6.0, Culture=neutral, PublicKeyToken=c07d481e9758c731">
+    <Reference Include="mscorlib">

```

Comment:
These versions and sing key tokens need to be here.

Reply:
I had problems updating to latest nugets so removed and added again and that's what i ended with. Seems to build ok.

Reply:
Had to manually update version for gitversion in project for it to update properly.